packages/odyssey-react-mui/src/labs/AppTile.tsx

```diff
@@ -85,106 +88,205 @@ type LoadingTileProps = {
-  display: "flex",
-  minHeight: odysseyDesignTokens.Spacing6,
-  position: "absolute",
-  right: odysseyDesignTokens.Spacing2,
-  top: odysseyDesignTokens.Spacing2,
-  gap: odysseyDesignTokens.Spacing1,
-  zIndex: 1,
+  shouldForwardProp: (prop) =>
+    prop !== "odysseyDesignTokens" && prop !== "variant",
+})<{ odysseyDesignTokens: DesignTokens; variant: AppTileProps["variant"] }>(
+  ({ odysseyDesignTokens, variant = "comfortable" }) => ({
+    alignItems: "center",
+    display: "flex",
+    minHeight: odysseyDesignTokens.Spacing6,
+    position: "absolute",
+    right: variant === "compact" ? 0 : odysseyDesignTokens.Spacing2,
+    top: variant === "compact" ? 0 : odysseyDesignTokens.Spacing2,
+    gap: odysseyDesignTokens.Spacing1,
+    zIndex: 1,
 
-  ["& > .MuiTypography-root:last-child"]: {
-    marginInlineEnd: odysseyDesignTokens.Spacing2,
-  },
-}));
+    ["& > .MuiTypography-root:last-child"]: {
+      marginInlineEnd: odysseyDesignTokens.Spacing2,
+    },
+  }),
+);
 
 const ContentContainer = styled("div", {
   shouldForwardProp: (prop) =>
-    prop !== "odysseyDesignTokens" && prop !== "hasTopSection",
-})<{ odysseyDesignTokens: DesignTokens; hasTopSection: boolean }>(
-  ({ odysseyDesignTokens, hasTopSection }) => ({
-    alignItems: "flex-start",
-    paddingBlockStart: hasTopSection ? odysseyDesignTokens.Spacing4 : 0,
-  }),
-);
+    prop !== "odysseyDesignTokens" &&
+    prop !== "hasTopSection" &&
+    prop !== "variant",
+})<{
+  odysseyDesignTokens: DesignTokens;
+  hasTopSection: boolean;
+  variant: AppTileProps["variant"];
+}>(({ odysseyDesignTokens, hasTopSection, variant }) => ({
+  alignItems: "flex-start",
+  paddingBlockStart: hasTopSection ? odysseyDesignTokens.Spacing4 : 0,
+  height: "100%",
+
+  [".MuiTypography-root"]: {
+    fontSize:
+      variant === "compact"
+        ? odysseyDesignTokens.TypographySizeSubordinate
+        : "auto",
+    textAlign: variant === "compact" ? "center" : "left",
+  },
+
+  ["& :last-child"]: {
+    marginBlockEnd: "0 !important",
+  },
+}));
 
 const ChildrenContainer = styled("div", {
   shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
 })<{ odysseyDesignTokens: DesignTokens }>(({ odysseyDesignTokens }) => ({
   ["&:not(:first-child)"]: {
-    marginBlockStart: odysseyDesignTokens.Spacing3,
+    marginBlockStart: odysseyDesignTokens.Spacing4,
+  },
+}));
+
+const StyledMuiCard = styled(MuiCard, {
+  shouldForwardProp: (prop) =>
+    prop !== "odysseyDesignTokens" &&
+    prop !== "isPlaceholder" &&
+    prop !== "isCompact",
+})<{
+  odysseyDesignTokens: DesignTokens;
+  isPlaceholder?: boolean;
+  isCompact: boolean;
+}>(({ odysseyDesignTokens, isPlaceholder = false, isCompact = false }) => ({
+  position: "relative",
+  boxShadow: "none",
+  overflow: "unset",
+  display: "flex",
+  justifyContent: isPlaceholder && isCompact ? "center" : "flex-start",
+  padding: odysseyDesignTokens.Spacing4,
+  borderWidth: odysseyDesignTokens.BorderWidthMain,
```

Comment:
Not a blocker by any means but why not just use the short hand `border` declaration here?